fusion-plugin-apollo/src/server.js

```diff
@@ -8,10 +8,13 @@
 
 /* eslint-env node */
 import {getDataFromTree as defaultGetDataFromTree} from '@apollo/react-ssr';
+import {UniversalEventsToken} from 'fusion-plugin-universal-events';
 import type {Logger} from 'fusion-tokens';
-
 import type {Element} from 'react';
 
+type ExtractReturnType = <V>(() => V) => V;
+type IEmitter = $Call<ExtractReturnType, typeof UniversalEventsToken>;
```

Comment:
Just following from some other package. Looks like there is no `UniversalEvents` flow type exported from `fusion-plugin-universal-events`.